core/utils.go

```diff
@@ -0,0 +1,11 @@
+package rocketmq
+
+import "fmt"
+
+func strJoin(str, key string, value interface{}) string {
```

Comment:
`fmt.Stringer` interface is better than `interface{}`

Reply:
value passed in maybe is `int` ~ : )

Reply:
GOOD